queue/driver_mongo.go

```diff
@@ -90,23 +87,19 @@ func newMongoGroupDriver(name string, opts MongoDBOptions, group string) (remote
 }
 
 // OpenNewMongoGroupDriver constructs and opens a new MongoDB driver instance
-// using the specified session. It is equivalent to calling
-// NewMongoGroupDriver() and calling driver.Open().
-func openNewMongoGroupDriver(ctx context.Context, name string, opts MongoDBOptions, group string, client *mongo.Client) (remoteQueueDriver, error) {
+// using the specified client. The returned driver does not take ownership of
```

Comment:
OpenNewMongoGroupDriver -> openNewMongoGroupDriver on the line above.

Reply:
Fixed.